README.md

```diff
@@ -1 +1,2 @@
-# jpn-lang-api
\ No newline at end of file
+# jpn-lang-api
+# http://localhost:8082/swagger-ui/index.html
```

Comment:
you should add the description, what is the use of this url?

Reply:
updated project name.